Add 2 devices to Tohoma cover

m4dmin opened the pull request:
## Description:
**Added two devices to tahoma:**
io:RollerShutterUnoIOComponent
io:ExteriorVenetianBlindIOComponent

**Related issue (if applicable):** fixes #14318

## Checklist:
  - [x] The code change is tested and works locally.

[ex-requir]: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/keyboard.py#L14
[ex-import]: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/keyboard.py#L54

m4dmin committed:
add 2 devices

io:RollerShutterUnoIOComponent
io:ExteriorVenetianBlindIOComponent

homeassistant commented:
Hi @m4dmin, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant#14321).

Once you do that we will be able to review and accept this pull request.

Thanks!

balloob commented on homeassistant/components/cover/tahoma.py:
```diff
@@ -81,5 +81,9 @@ def stop_cover(self, **kwargs):
             self.apply_action('setPosition', 'secured')
         elif self.tahoma_device.type == 'rts:BlindRTSComponent':
             self.apply_action('my')
+        elif self.tahoma_device.type == 'io:RollerShutterUnoIOComponent':
```
Since it's the same command, please use a single if check:

```python
elif self.tahoma_device.type in ('bla', 'bli', 'blub'):
```

m4dmin committed:
add 2 devices

m4dmin committed:
Update tahoma.py

m4dmin committed:
Fix hounci-bot violation

m4dmin committed:
Fixed Travis CI build failure

./homeassistant/components/cover/tahoma.py:83:13: E125 continuation line with same indent as next logical line

m4dmin committed:
Fixed Travis CI build failure

E125 continuation line with same indent as next logical line

m4dmin committed:
Fixed Travis CI build failure

E127 continuation line over-indented for visual indent

fabaff committed:
Fix indent

fabaff committed:
Change check

fabaff reviewed (approved):
:bird:

m4dmin commented:
Many thanks for your help!
